frontend/src/metabase/lib/dom.js

```diff
@@ -194,31 +194,8 @@ function getTextNodeAtPosition(root, index) {
   };
 }
 
-// https://davidwalsh.name/add-rules-stylesheets
-const STYLE_SHEET = (function () {
-  // Create the <style> tag
-  const style = document.createElement("style");
-
-  // WebKit hack :(
-  style.appendChild(document.createTextNode("/* dynamic stylesheet */"));
-
-  // Add the <style> element to the page
-  document.head.appendChild(style);
-
-  return style.sheet;
-})();
-
 export function addCSSRule(selector, rules, index = 0) {
-  if ("insertRule" in STYLE_SHEET) {
-    const ruleIndex = STYLE_SHEET.insertRule(
-      selector + "{" + rules + "}",
-      index,
-    );
-    return STYLE_SHEET.cssRules[ruleIndex];
-  } else if ("addRule" in STYLE_SHEET) {
-    const ruleIndex = STYLE_SHEET.addRule(selector, rules, index);
-    return STYLE_SHEET.rules[ruleIndex];
-  }
+  // noop to test emotion changes
```

Comment:
I'm not sure how this is/will be used to test emotion changes.

I think it's a noop so that we can work on `LineAreaBarChart` and `Logs` later, no?

Reply:
Yep